src/api/x11/window.rs

```diff
@@ -718,6 +718,16 @@ impl Window {
     }
 
     #[inline]
+    pub fn get_xlib_screen_id(&self) -> *mut libc::c_void {
```

Comment:
Ideally this should be `std::os::raw::c_void` and not `libc::c_void`, but since an existing method also uses `libc` I'm ok with it.